.github/workflows/main.yml

```diff
@@ -0,0 +1,25 @@
+name: ShellCheck
+
+on: [push]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Install ShellCheck
+      run: sudo apt-get install shellcheck
```

Comment:
Is this the best practice for installing packages?

Reply:
Seems to be? If it works and is performant enough for you, I'd say 👍